app-misc/dtc/dtc-2.6.ebuild

```diff
@@ -0,0 +1,120 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop udev unpacker xdg-utils
+
+DESCRIPTION="KryoFlux Host Software"
+HOMEPAGE="https://www.kryoflux.com"
+SRC_URI="https://www.kryoflux.com/download/kryoflux_${PV}_linux.tar.bz2
+	https://www.kryoflux.com/kryoflux-ui.jar"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="SPS"
+SLOT="0"
+IUSE="demos doc fast-firmware java static"
+
+RDEPEND="dev-libs/spsdeclib
+	virtual/libusb:1
+	virtual/udev
+	java? ( virtual/jre )"
+
+RESTRICT="bindist mirror strip"
+
+S="${WORKDIR}/kryoflux_${PV}_linux"
+
+DOCS=( "DONATIONS.txt" "RELEASE.txt" "README.linux" )
+
+QA_PREBUILT="usr/bin/dtc
+	usr/bin/dtc.static"
+
+src_unpack() {
+	unpack kryoflux_${PV}_linux.tar.bz2
+
+	# Use updated *.jar for compatibility with >= Java 9
+	if use java; then
+		cp "${DISTDIR}"/kryoflux-ui.jar "${S}" || die
+	fi
+
+	# Extract kryoflux-ui.jar to get a logo for the meny entry
+	unpack_zip kryoflux-ui.jar
```

Comment:
Do you really need the special unpacker unpack_zip instead of default unpack? Also missing dep on unzip.

Also do you really need to do this unconditionally? It looks like the image is only used with USE=java.